src/data/faq_de.json

```diff
@@ -2041,7 +2041,7 @@
                                 "anchor": "international_phone_numbers",
                                 "active": false,
                                 "textblock": [
-                                    "<div class='update'>Die Informationen in diesem FAQ-Eintrag stehen im Zusammenhang mit Funktionen oder beschreiben Funktionen der App, die seit dem 1. Mai 2023 nicht mehr zur Verfügung stehen, und sind daher nicht mehr aktuell. Weitere Informationen finden Sie hier: <a href='#ramp_down' target='_blank'>Was ist der Hintergrund für das Ende der Entwicklung der CWA?</a></div>",
+                                    "<div class='update'>Der Betrieb der technischen Hotline ist seit dem 1. Juni 2023 eingestellt worden. Die Informationen in diesem FAQ-Eintrag sind daher nicht mehr aktuell. Weitere Informationen finden Sie hier: <a href='#ramp_down' target='_blank'>Was ist der Hintergrund für das Ende der Entwicklung der CWA?</a></div>",

```

Comment:
```suggestion
                                    "<div class='update'>Der Betrieb der technischen Hotline ist am 1. Juni 2023 eingestellt worden. Die Informationen in diesem FAQ-Eintrag sind daher nicht mehr aktuell. Weitere Informationen finden Sie hier: <a href='#ramp_down' target='_blank'>Was ist der Hintergrund für das Ende der Entwicklung der CWA?</a></div>",
```

Reply:
or use the same wording as for the TAN hotline:

"Der Betrieb der technischen Hotline wurde zum 1. Juni 2023 eingestellt."